independent-publisher-connectors/rSpaceX/apiDefinition.swagger.json

```diff
@@ -0,0 +1,11148 @@
+                      "items": {
+                        "type": "string"
+                      },
+                      "description": "The large images.",
+                      "title": "Large Images."
+                    }
+                  },
+                  "description": "The images.",
+                  "title": "images"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "The name.",
+                  "title": "Name"
+                },
+                "full_name": {
+                  "type": "string",
+                  "description": "The full name.",
+                  "title": "Full Name"
+                },
+                "locality": {
+                  "type": "string",
+                  "description": "The locality.",
+                  "title": "Locality"
+                },
+                "region": {
+                  "type": "string",
+                  "description": "The region.",
+                  "title": "Region"
+                },
+                "latitude": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "The latitude.",
+                  "title": "Latitude"
+                },
+                "longitude": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "The longitude.",
+                  "title": "Longitude"
+                },
+                "launch_attempts": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The launch attempts.",
+                  "title": "Launch Attempts"
+                },
+                "launch_successes": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The launch successes.",
+                  "title": "Launch Successes"
+                },
+                "rockets": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "The rockets.",
+                  "title": "Rockets"
+                },
+                "timezone": {
+                  "type": "string",
+                  "description": "The timezone.",
+                  "title": "Timezone"
+                },
+                "launches": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "The launch identifiers.",
+                  "title": "Launches"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "The current status.",
+                  "title": "Status"
+                },
+                "details": {
+                  "type": "string",
+                  "description": "The details.",
+          
```

Comment:
@troystaylor Can you please valid description value.

Reply:
Changed to "The launchpad details."

Reply:
@troystaylor I can not see the update made to this property. Can you please revisit, thanks

Reply:
Updated